lib/symbol-store.coffee

```diff
@@ -174,6 +174,7 @@ class SymbolStore
 
     for bufferRow in [bufferRange.start.row..bufferRange.end.row] by 1
       tokenizedLine = tokenizedLines[bufferRow]
```

Comment:
I wonder if it will introduce inconsistent states, however, and lead to false positives and negatives.